packages/preprocess/src/process-queries.cjs

```diff
@@ -12,31 +12,31 @@ const createDefaultProps = function (filename, componentDevelopmentMode, duckdbQ
 		const valid_ids = Object.keys(duckdbQueries).filter((queryId) =>
 			queryId.match('^([a-zA-Z_$][a-zA-Z0-9d_$]*)$')
 		);
-		queryDeclarations += `
-            import debounce from 'debounce';
-            import { browser } from '$app/environment';
-			import {profile} from '@evidence-dev/component-utilities/profile';
-			
-			// partially bypasses weird reactivity stuff with \`select\` elements
-			function data_update(data) {
-				${valid_ids.map((id) => `${id} = data.${id} ?? [];`).join('\n')}
-			}
-
-			$: data_update(data);
 
+		const queryStores = valid_ids.map(
+			(id) => `
+		const _${id} = new QueryStore(
```

Comment:
```suggestion
		$: _${id} = new QueryStore(
```
It's possible that 
```js
\`${duckdbQueries[id].replaceAll('`', '\\`')}\`
```
has interpolated variables in it, so this has to be reactive to that